x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -36576,7 +36576,6 @@
     "xpack.security.management.editRole.featureTable.cannotCustomizeSubFeaturesTooltip": "La personnalisation des privilèges de sous-fonctionnalité est une fonctionnalité soumise à abonnement.",
     "xpack.security.management.editRole.featureTable.customizeSubFeaturePrivilegesSwitchLabel": "Personnaliser les privilèges des sous-fonctionnalités",
     "xpack.security.management.editRole.featureTable.featureAccordionSwitchLabel": "{grantedCount}/{featureCount} {featureCount, plural, one {fonctionnalité accordée} other {fonctionnalités accordées}}",
-    "xpack.security.management.editRole.featureTable.featureVisibilityTitle": "Personnaliser les privilèges des fonctionnalités",
```

Comment:
IIRC we should not commit translation changes as part of the PR

Reply:
These are automatic changes done via `node ./scripts/i18n_check.js --fix`, which is needed for a green build.

Reply:
Got it! Thanks